lca-chimevc-stack/cloudformation-templates/chime-vc-siprec.yaml

```diff
@@ -89,8 +97,13 @@ Resources:
       Handler: index.handler
       Runtime: python3.11
       Timeout: 300
+      Environment:
```

Comment:
Better to pass this in the Properties of the CustomResource rather than as a Lambda ENV var.  Presumably you want the function to be invoked during a stack update if the value of the parameter has changed.. unless this is included as a Custom Resource property, the Lambda function will not be actually invoked by CloudFormation during updates.. 
Please thoroughly test stack updates where you toggle this param, and confirm results.